### PR TITLE
Honk Bullet

### PR DESCRIPTION
**Pull request by Robustin**
Very important PR.

Bullet only available through admin memes, but I can't stress enough how much our code needs a long-range meme bullet that doesn't kill people.

Edit:

Added Staff of the Honkmother, also admin-only. Fires the projectile at a limited rate. Quick merge now plz.

### CODE REVIEW

File: code/modules/projectiles/projectile/bullets.dm [hunk 1]
```diff
@@ -124,6 +124,16 @@
 	damage = 10
 	weaken = 4
 	stun = 4
+	
+/obj/item/projectile/bullet/honker
+	damage = 0
+	weak = 5
```
Comment (KorPhaeron) [inline code/modules/projectiles/projectile/bullets.dm]: weaken

Comment (Robustin) [inline code/modules/projectiles/projectile/bullets.dm]: 2 late, words no gud

**Comment by Shapsy** [comment]
Shouldn't you add magazines for the guns as well for ease of use

**Comment by Robustin** [comment]
This gets var/edited in, so its just either going to be shot directly from the item (turrets, mechs, simple animals) which only uses the projectile.

Or its going to be edited into an existing gun, which will already has its own existing magazine system that's pretty finnicky about what you can stuff in there, so its always much easier to go to the source and say "shoot clown bullets" instead of whatever it was going to fire.

But the chief issue isn't the smaller guns, its those machines and such that ONLY shoot projectiles and I've found that when you're in a situation where you can ONLY choose the projectile and can't edit the vars in advance (with guns you can just spawn all the ammo you intend to use then mass modify), you have pretty limited options.

File: code/modules/projectiles/projectile/bullets.dm [hunk 2]
```diff
@@ -124,6 +124,16 @@
 	damage = 10
 	weaken = 4
 	stun = 4
+	
+/obj/item/projectile/bullet/honker
+	damage = 0
+	weaken = 5
+	stun = 5
+	forcedodge = 1
+	hitsound = 'sound/items/bikehorn.ogg'
```
Comment (KazeEspada) [inline code/modules/projectiles/projectile/bullets.dm]: Not an airhorn?

Comment (Robustin) [inline code/modules/projectiles/projectile/bullets.dm]: I fall into the conservative wing of the Clown coalition.

If I put it into guns, the firesound is always an airhorn.

Comment (KazeEspada) [inline code/modules/projectiles/projectile/bullets.dm]: Also, theres a var called nodamage you need to set.

**Comment by KorPhaeron** [comment]
Should at least make a wand or something that fires it

**Comment by Robustin** [comment]
@KorPhaeron 

Didn't want to make a larger footprint than necessary for memes, but this game does sorely lack fun loot that doesn't immediately break the game.

**Comment by Impisi** [comment]
I'm going to shill for table crafting and reccomend you add in a recipe for making a honk shotgun shell. Maybe a banana, 5 metal, some wires and a bikehorn as the tool to put it together?

File: code/modules/projectiles/projectile/bullets.dm (hunk 2 above)
Comment (Robustin) [inline code/modules/projectiles/projectile/bullets.dm]: If its already set to 0, why?

Comment (optimumtact) [inline code/modules/projectiles/projectile/bullets.dm]: because our code is bad

Comment (KazeEspada) [inline code/modules/projectiles/projectile/bullets.dm]: Because #GUN CODE

**Comment by Robustin** [comment]
Before I start pushing this for getting merged, I just wanted to throw out a question. I'd like to make the banana bullet icon a little smaller so that it actually looks closer to a projectile and isn't so shoddy looking. I know botany makes tiny bananas but I can't figure out if that's something I could incorporate into the icon here. 

What would be the easiest way to accomplish this?

File: code/modules/projectiles/projectile/bullets.dm [hunk 3]
```diff
@@ -124,6 +124,21 @@
...
+	weaken = 5
+	stun = 5
+	forcedodge = 1
+	nodamage = 1
+	hitsound = 'sound/items/bikehorn.ogg'
+	icon = 'icons/obj/hydroponics/harvest.dmi'
+	icon_state = "banana"
+	range = 200
+	
+/obj/item/projectile/bullet/meteorshot/New()
```
Comment (KorPhaeron) [inline code/modules/projectiles/projectile/bullets.dm]: ?

Comment (Robustin) [inline code/modules/projectiles/projectile/bullets.dm]: HNNNGH

File: code/modules/projectiles/projectile/bullets.dm [hunk 4]
```diff
@@ -124,6 +124,21 @@
 	damage = 10
 	weaken = 4
 	stun = 4
+	
+/obj/item/projectile/bullet/honker
+	damage = 0
+	weaken = 5
+	stun = 5
+	forcedodge = 1
```
Comment (phil235) [inline code/modules/projectiles/projectile/bullets.dm]: You're aware this means the bullet will pass through walls, machines and mobs, right? And with a range of 200 you're going to end up hearing that bikehorn sound a lot.

Comment (Robustin) [inline code/modules/projectiles/projectile/bullets.dm]: Precisely. Bikehorn is only a local sound and theres not exactly a ton of mobs on the station, youll get 2-3 honks per shot if you're lucky.

File: code/modules/projectiles/ammunition/special.dm [hunk 5]
```diff
@@ -31,6 +31,10 @@
 	projectile_type = pick(typesof(/obj/item/projectile/magic))
 	..()
 
+/obj/item/ammo_casing/magic/honk
+	projectile_type = /obj/item/projectile/bullet/honker
+	..()
+
```
Comment (KorPhaeron) [inline code/modules/projectiles/ammunition/special.dm]: What is this nonsense

**Pull request merged**